### PR TITLE
Terraform locals and iam policy document

### PR DESCRIPTION
**Pull request by Chufle**
- added locals for s3 bucket photos in "locals.tf"
- new file "s3buckets.tf", creation of photos bucket inside, instead of inside of "lambda_photo_checkin.tf"
- new file "iam_policy_document.tf", which is referenced from "iam_for_lambda.tf"

### CODE REVIEW

**Review by melectronx — COMMENTED** [review]
LGTM

File: infrastructure/iam_policy_document.tf [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+        "dynamodb:DescribeTable",
+        "dynamodb:Get*",
+        "dynamodb:Query",
+        "dynamodb:Scan",
+        "dynamodb:DeleteItem",
+        "dynamodb:UpdateItem",
+        "dynamodb:PutItem"
+    ]
+    resources = [
+        "arn:aws:dynamodb:us-west-2:${data.aws_caller_identity.current.account_id}:table/kutter-table"
```
Comment (fabianschmauder) [inline infrastructure/iam_policy_document.tf:34]: this is the arn from the resource. 
You can directly refrerence the arn. 

I think like:
`aws_dynamodb_table.dynamodb-kutter-table.arn`

**Review by fabianschmauder — COMMENTED** [review]
nice just the reference

File: infrastructure/iam_policy_document.tf (hunk 1 above)
Comment (Chufle) [reply infrastructure/iam_policy_document.tf:34]: I tried it but like this I get an error:
```
╷
│ Error: error updating IAM policy arn:aws:iam::826643085812:policy/nfish-des-pol-lambda_s3_dynamodb: MalformedPolicyDocument: Resource aws_dynamodb_table.dynamodb-kutter-table.arn must be in ARN format or "*".
│       status code: 400, request id: b789e877-cf7b-4787-90a6-afbf38c5ebad
│ 
│   with aws_iam_policy.nfish-des-pol-lambda_s3_dynamodb,
│   on iam_for_lambda.tf line 1, in resource "aws_iam_policy" "nfish-des-pol-lambda_s3_dynamodb":
│    1: resource "aws_iam_policy" "nfish-des-pol-lambda_s3_dynamodb" {
│ 
╵
```

Comment (Chufle) [reply infrastructure/iam_policy_document.tf:34]: now it works

**Pull request merged**